plugins/module_utils/network/nxos/facts/legacy/base.py

```diff
@@ -162,6 +162,47 @@ def populate(self):
                 self.facts["memtotal_mb"] = self.parse_memtotal_mb(data)
                 self.facts["memfree_mb"] = self.parse_memfree_mb(data)
 
+        data = None
+        data = self.run("show processes cpu | include CPU")
```

Comment:
Can we just the `| json` or `| json-pretty` filter here? That would hopefully help reduce the amount of regexes needed.

Reply:
I used json pretty it gave no output, checking it again!